assignments/assignment02.rb

```diff
@@ -25,11 +35,30 @@ def to_sentence(ary)
 
 # implement methods "mean", "median" on Array of numbers
 def mean(ary)
-  # your implementation here
+  # add all the numbers together
+  sum = ary.reduce {|i, acc| i + acc }
+
+  # divide by the length, convert to floating point
+  mean = sum/ary.length.to_f
+  return mean
 end
 
 def median(ary)
-  # your implementation here
+  # find the length of the array 
+  len = ary.length
+  
+  # sort the array
+  ary.sort!
```

Comment:
I discourage using the ! methods. Try to find solutions that don't use these. You could do "sorted_ary = ary.sort" and then use sorted_ary.